web/src/components/DandiAppBar.vue

```diff
@@ -105,6 +161,21 @@ export default {
       return 'NA';
     },
   },
-  methods: mapActions(['logout', 'selectSearchResult']),
+  methods: {
+    async register_dandiset() {
+      const { name, description } = this;
+      const { status, data } = await this.girderRest.post(`dandi`,
+          stringify({ name: name, description: description}));
+
+      if (status === 200) {
+        this.name = '';
+        this.description = '';
+        this.$router.push({
```

Comment:
This is the place that will not refresh the view, because the same component is being reused.